packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -161,97 +222,102 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
-                        setErrorMessage={setErrorMessage}
-                        setSubmitForm={setSubmitForm}
-                        setPageFooterParent={setPageFooterParent}
-                    />
-                )}
-            </MobileFullPageModal>
-        );
-    }
-    if (should_show_popup) {
-        return (
-            <Modal
-                className='buy-sell__modal'
-                height={table_type === buy_sell.BUY ? 'auto' : '649px'}
-                width='456px'
-                is_open={should_show_popup}
-                title={generateModalTitle(formik_ref, my_profile_store, table_type, selected_ad)}
-                portalId={general_store.props.modal_root_id}
-                toggleModal={onCancel}
-            >
-                {/* Parent height - Modal.Header height - Modal.Footer height */}
-                <ThemedScrollbars height={table_type === buy_sell.BUY ? '100%' : 'calc(100% - 5.8rem - 7.4rem)'}>
-                    <Modal.Body className='buy-sell__modal--layout'>
-                        {table_type === buy_sell.SELL && is_account_balance_low && <LowBalanceMessage />}
-                        {!!error_message && <BuySellFormError />}
-                        {my_profile_store.should_show_add_payment_method_form ? (
-                            <AddPaymentMethodForm formik_ref={formik_ref} should_show_separated_footer />
-                        ) : (
-                            <Form
-                                advert={selected_ad}
-                                handleClose={onCancel}
-                                handleConfirm={onConfirmClick}
-                                setIsSubmitDisabled={setIsSubmitDisabled}
-                                setErrorMessage={setErrorMessage}
-                                setSubmitForm={setSubmitForm}
-                            />
-                        )}
-                    </Modal.Body>
-                </ThemedScrollbars>
-                {!my_profile_store.should_show_add_payment_method_form && (
-                    <Modal.Footer has_separator>
-                        {my_profile_store.should_show_add_payment_method_form ? null : (
+    return (
+        <React.Fragment>
+            <MarketRateChangeErrorModal
+                is_open={show_market_rate_change_error_modal}
+                closeModal={() => setShowMarketRateChangeErrorModal(false)}
+                setShouldShowPopup={setShouldShowPopup}
+            />
+            <MobileWrapper>
+                <MobileFullPageModal
+                    body_className='buy-sell__modal-body'
+                    className='buy-sell__modal'
+                    height_offset='80px'
+                    is_flex
+                    is_modal_open={should_show_popup}
+                    page_header_className='buy-sell__modal-header'
+                    page_header_text={generateModalTitle(formik_ref, my_profile_store, table_type, selected_ad)}
```

Comment:
Same here.

Reply:
I checked in this case `page_header_text` has a string type thou